project/FrontEnd/collaborative_science_platform/lib/providers/settings_provider.dart

```diff
@@ -0,0 +1,56 @@
+import 'dart:convert';
+import 'package:collaborative_science_platform/utils/constants.dart';
+import 'package:flutter/material.dart';
+import 'package:http/http.dart' as http;
+
+class SettingsProvider with ChangeNotifier {
+  String token = "cd018223f0564aa0f66af496d754c819494de87f"; //TEST
+  Future<void> changePassword(String oldPass, String newPass) async {
+    final Map<String, String> header = {
+      "Accept": "application/json",
+      "content-type": "application/json",
+      'Authorization': 'Token $token',
```

Comment:
And here we should use user.token 

Before merging let's fix this. 

I know instead you used dummy token but still this is not a good usage. Instead setting user.token to something dummy is a better approach. 

For now don't add anything dummy, just add user.